pkg/kwok/controllers/node_controller.go

```diff
@@ -98,8 +99,9 @@ type NodeController struct {
 	preprocessChan                        chan *corev1.Node
 	playStageParallelism                  uint
 	lifecycle                             resources.Getter[Lifecycle]
-	delayQueue                            queue.DelayingQueue[resourceStageJob[*corev1.Node]]
+	delayQueue                            queue.WeightDelayingQueue[resourceStageJob[*corev1.Node]]
```

Comment:
We should add the same support for Pod and Stage

Reply:
Hi @wzshiming  Yes. I will change other controllers as well If there are no obvious problems with this version.